Userland/Libraries/LibWeb/HTML/Navigable.cpp

```diff
@@ -1409,6 +1409,10 @@ WebIDL::ExceptionOr<void> Navigable::navigate(NavigateParams params)
                 }
                 finalize_a_cross_document_navigation(*this, to_history_handling_behavior(history_handling), history_entry);
             });
+
+            if (is_top_level_traversable() && url != history_entry->url) {
+                active_browsing_context()->page().client().page_did_start_loading(history_entry->url, true);
```

Comment:
I believe a recent refactor by Andreas changed the way that we access the Page normally, and it's not via the BrowsingContext anymore. I can't remember where it's supposed to be gotten from off the top of my head though. perhaps the Realm? or the navigable/traversable navigable will have a cached GCPtr to it?